templates/unbound.conf.erb

```diff
@@ -182,3 +182,7 @@ server:
 <% @custom_server_conf.each do |c| -%>
   <%= c %>
 <% end -%>
+<% if @python -%>
+python:
```

Comment:
Yeah, the docs are a little weak but this is correct: 
https://github.com/jedisct1/unbound/blob/master/doc/example.conf.in#L737